calculator/main.py

```diff
@@ -58,18 +80,25 @@ def get_history(calc):
         ie:
         ('2016-05-20 12:00:00', 'add', (1, 2), 3),
     """
-    pass
+    return calc['history']
+  
 
 
 def reset_history(calc):
     """
     Resets the calculator history back to an empty list.
     """
-    pass
+    calc['history'] = []
+
 
 
 def repeat_last_operation(calc):
     """
     Returns the result of the last operation executed in the history.
     """
-    pass
+    if len(calc['history']) == 0:
+        return None
+    
+    _,_,_,return_val = calc['history'][-1]
+    return return_val
```

Comment:
That would return the whole tuple not just the total of the last operation. We could have just done 

`return_val = calc['history'][-1]`
`return return_val[3]`
or just
`return calc['history'][-1][3]`